src/main/java/uk/gov/hmcts/reform/sscs/personalisation/WithRepresentativePersonalisation.java

```diff
@@ -23,14 +24,21 @@
 
     public Map<String, String> setRepresentativeName(Map<String, String> personalisation, SscsCaseData sscsCaseData) {
         if (isValidReps(sscsCaseData.getAppeal().getRep())) {
-            personalisation.put(AppConstants.REPRESENTATIVE_NAME, String.format("%s %s",
-                    sscsCaseData.getAppeal().getRep().getName().getFirstName(),
-                    sscsCaseData.getAppeal().getRep().getName().getLastName()));
+            personalisation.put(AppConstants.REPRESENTATIVE_NAME, 
+                    getDefaultName(sscsCaseData.getAppeal().getRep().getName(),
+                    "Sir / Madam"));
         }
         return personalisation;
     }
 
     private boolean isValidReps(Representative representative) {
-        return null != (representative) && null != representative.getName();
+        if (representative == null) {
+            return false;
+        }
+
+        Name repName = representative.getName();
+
+        return (null != repName && StringUtils.isNotBlank(repName.getFirstName()) && StringUtils.isNotBlank(repName.getLastName()))
```

Comment:
You might want to consider using the existing uk.gov.hmcts.reform.sscs.service.NotificationUtils#hasRepresentative method as part of this assertion.

Reply:
I'll see if I can use that. On initial inspection is seems like NotificationUtils::hasRepresentative() wants a case data wrapper which I'm not sure is available down at this level. I'll see if I can perhaps combine a call to hasRep() further up so it's like `NotificationUtils.hasRepresentative() && isValidReps()` or something like that.

Reply:
Incorporated `hasRepresentative()` at the point which `isValidReps()` is called, as per previous comment.